airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -58,6 +60,11 @@ public class MySqlSource extends AbstractJdbcSource<MysqlType> implements Source
       "useSSL=true",
       "requireSSL=true",
       "verifyServerCertificate=false");
+  private final Set<MysqlType> allowedCursorTypes = Set.of(BIT, BOOLEAN, TINYINT, TINYINT_UNSIGNED, SMALLINT,
```

Comment:
BIT, BOOLEAN, ENUM, SET, TINYBLOB, BLOB, MEDIUMBLOB, LONGBLOB, BINARY, VARBINARY - not sure that they are reliable data types for cursors.

Reply:
That is an interesting question cause the list of possible cursor fields was taken from 
https://github.com/airbytehq/airbyte/blob/3894134d1115b7c9ba181684f1aba66bec500a63/airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSourceOperations.java#L114

I a see from implementation all this types can be used as cursors  

/**
   * Set the cursor field in incremental table query.
   */
  void setStatementField(final PreparedStatement preparedStatement,
                         final int parameterIndex,
                         final SourceType cursorFieldType,
                         final String value)
      throws SQLException;